mandatory/4-tax.js

```diff
@@ -5,7 +5,9 @@
   Sales tax is 20% of the price of the product.
 */
 
-function calculateSalesTax() {}
+function calculateSalesTax(amount) {
+  return amount = ((amount * 0.2) + (amount));
```

Comment:
Good implementation, but can it be expressed more concisely with fewer maths operations?

Reply:
return amount = (1.2 * amount)